kotlin/trace-utility-network/README.md

```diff
@@ -26,7 +26,7 @@ Tap on one or more features while 'Add starting locations' or 'Add barriers' is
 9.  If an edge, set its `FractionAlongLine` property using `GeometryEngine.fractionAlongEdge`.
 10. Add this `UtilityElement` to a collection of starting locations or barriers.
 11. Create `TraceParameters` with the selected trace type along with the collected starting locations and barriers (if applicable). 
-12. Set the `TraceParameters.traceConfiguration` with the utility tier's `TraceConfiguration` property.
+12. Set the `TraceParameters.defaultTraceConfiguration` with the utility tier's `TraceConfiguration` property.
```

Comment:
this doesn't need updated: the traceconfiguration method on TraceParameters has not been deprecated :)

Reply:
Ahh good spot, thanks!